### PR TITLE
homework 7

### PR DESCRIPTION
**Pull request by olyatmchshn**

### CODE REVIEW

File: homeworks/olha.tymchyshyn_olyatmchshn/7-arrays-loops/arrays.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+function maxNumberOfArr(array) {
+    let maxNum = array[0];
+    for (let i = 0; i < array.length; i++) {
+        if (array[i] > maxNum) {
+            maxNum = array[i];
+        }
+    }
+    return maxNum;
+    // write your code here and instead of array return maxNumber
```
Comment (vladyslav-yermolin-moc) [inline homeworks/olha.tymchyshyn_olyatmchshn/7-arrays-loops/arrays.js:15]: Please remove these comments

File: homeworks/olha.tymchyshyn_olyatmchshn/7-arrays-loops/arrays.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
+const arrayOne = [3, 0, -5, 1, 44, -12, 3, 0, 0, 1, 2, -3, -3, 2, 1, 4, -2, -3, -1];
+const arrayTwo = [-1, -8, -2];
+const arrayThree = [1, 7, 3];
+const arrayFour = [1, undefined, 3, 5, -3];
+const arrayFive = [1, NaN, 3, 5, -3];
+
+function maxNumberOfArr(array) {
+    let maxNum = array[0];
+    for (let i = 0; i < array.length; i++) {
```
Comment (vladyslav-yermolin-moc) [inline homeworks/olha.tymchyshyn_olyatmchshn/7-arrays-loops/arrays.js:9]:
```suggestion
    for (let i = 1; i < array.length; i++) {
```

File: homeworks/olha.tymchyshyn_olyatmchshn/7-arrays-loops/arrays.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+console.log(onlyNegativeNumbers(arrayOne)); // [-5, -12, -3, -3, -2, -3, -1]
+console.log(onlyNegativeNumbers(arrayTwo)); // [-1, -8, -2]
+console.log(onlyNegativeNumbers(arrayThree)); // []
+console.log(onlyNegativeNumbers(arrayFour)); //  [-3]
+console.log(onlyNegativeNumbers(arrayFive)); // [-3]
+
+function onlyPositiveNumbers(array) {
+    const positiveNumbers = array.filter((number) => number > 0);
```
Comment (vladyslav-yermolin-moc) [inline homeworks/olha.tymchyshyn_olyatmchshn/7-arrays-loops/arrays.js:71]: image
Common approach; you set something to variable, if you like to use it twice at least.
You can use `=>` single line function

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]

**Comment by vladyslav-yermolin-moc** [comment]
Gosh, I hope you're all right 😳
image

**Comment by olyatmchshn** [comment]
don't worry, it's ok for me to go to bed at 5-6 am😁

**Review by vladyslav-yermolin-moc — APPROVED** [review]

**Pull request merged**